figure.html

```diff
@@ -311,13 +487,24 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
 </li>
 
 <li>
-<p class="first">Click <span class="bold">Export PDF</span> to create an fully editable vector-based PDF of your figure.</p>
-<p>The PDF created is added to the images used in the figure as an attachment.</p>
+<p class="first">Click <span class="bold">Export PDF</span> and select a format from the drop-down to export your figure.</p>
+
+<p>The PDF version is vector-based and therefore fully editable after export.</p>
```

Comment:
We need to emphasis that selecting a format and actually starting the figure export are 2 different actions/clicks.
E.g (this is a bit long, but it's all useful info):  "Click Export PDF to create a figure for download. When this is complete a download button will become available. You can select different export options from the drop-down menu: Choosing "TIFF" will export the figure as a TIFF image at 300 dpi. The 'with images' options will create a zip file that includes the figure and all the individual figure panels as separate images. This will include the different processing stages of zooming, cropping, rotating, resampling etc to act as a record of the figure generation steps. NB: All exported figures and zips will be stored on the server and attached to the images that are in the figure. The zips may become large if they include component images, but can be safely deleted after download if desired.